velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -55,9 +55,10 @@ int main(int argc, char** argv) {
 
   // The following list are the Spark UDFs that hit issues
   // For rlike you need the following combo in the only list:
-  // rlike, md5 and upper
+  // rlike, md5 and upper, might_contain argument is a serialized bloom filter,
+  // it cannot accept random data.
   std::unordered_set<std::string> skipFunctions = {
-      "regexp_extract", "rlike", "chr", "replace"};
+      "regexp_extract", "rlike", "chr", "replace", "might_contain"};
```

Comment:
Let's figure out how to avoid adding this new function to the skip list.

Reply:
Can we consider VELOX_USER_CHECK exception as normal case which can pass fuzzer test? This may be the illegal user input

Reply:
This is already the case. Are you still seeing fuzzer failures? Can you share the output?